nx/lib/nx.ex

```diff
@@ -7842,6 +7842,70 @@ defmodule Nx do
     impl!(tensor).sort(tensor, tensor, axis: axis, comparator: comparator)
   end
 
+  @doc """
+  Solve the equation a x = b for x, assuming a is a triangular matrix.
+
```

Comment:
What do you think about adding a section on the valid options?

Reply:
Sounds good.